### ISSUE TITLE
Build server notifications - feature request

### ISSUE THREAD

**Issue by codenameone** [codenameone/CodenameOne#550]
Original [issue 550](https://code.google.com/p/codenameone/issues/detail?id=550) created by codenameone on 2013-02-20T13:16:01.000Z:

in the webpage: http://www.codenameone.com/build-server.html
Once a build is finished, please make a sound (beep or something) to indicate that a build has finished.

Allow sending an tag (string) with the build, so that once the build is finished send an email with this string (i.e., &quot;MyApplication - Build number 40&quot;)

Allow to send a build to both android and iOS simultaneously.

Thanks,
Ram

**Comment by codenameone** [comment]
Closing old Google code issues which are no longer relevant by now, we will offer better notifications in the near future
